index.html

```diff
@@ -560,15 +564,18 @@
         <div dfn-for="MediaTrackSettings">
           <p>
             The <dfn><code>focalLength</code></dfn> dictionary member
-            represents the <a>depth map</a>'s <a>focal length</a>.
+            represents the <a>depth map</a>'s <a>focal length</a>. It is a
+            <a>non-constrainable property</a>.
```

Comment:
Having things that only show up in getSettings violates the Constrainable pattern, so unless we have a good reason, I think we should mirror facingMode. Is there a downside?